firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Bytes.txt

```diff
@@ -0,0 +1,59 @@
+Scenario:The Util get_byte function correctly gets bytes from the given position
```

Comment:
This test needs to specify `WHEN`+`THEN` pairs for calling `get_byte` with byte indices which are out-of-range.